### ISSUE TITLE
change-licence listed twice in virtualmin CLI

### ISSUE THREAD

**Issue by shoulders** [virtualmin/virtualmin-gpl#793]
|  SYSTEM INFORMATION||
|----------------------|---------------------------|
|  OS type and version |     Ubuntu Linux 22.04.4     |
|  Webmin version      |     2.111      |
|  Usermin version     |     2.010     |
|  Virtualmin version  |     7.10.0       |
|  Theme version       |     21.10      |
|  Package updates     |     All installed packages are up to date   |

The following comand is listed twice when you run the terminal command `virtualmin` or `virtualmin help`.

I know this is a small thing. 😄 

`change-licence            Change a system's Virtualmin license key`

![image](https://github.com/virtualmin/virtualmin-gpl/assets/319997/d34636ff-2df0-4f1b-90ec-ce86e66c56ff)

**Comment by iliajie** [comment]
Looks like we're diving into the "license" vs. "licence" debate—let the spelling battle begin!

**Comment by shoulders** [comment]
I did not even know they were spelt different. 😄

**Comment by iliajie** [comment]
There are several words with different spellings between American and British English, such as "Practice" (US) vs "Practise" (UK), "Defense" (US) vs "Defence" (UK), "Analyze" (US) vs "Analyse" (UK), and "Color" (US) vs "Colour" (UK), among others.

Addressing this issue in a sensible way is challenging. Personally, I would favor US spelling because it is more commonly recognized globally, often shorter, and simpler. I believe it's not practical to support both.

**Comment by shoulders** [comment]
However you spell color/colour use that language format. Which I am guessing is US.

I am biased because I am here in Blighty (a.k.a UK)

Being British I am use to the dodgy 😄 USA spellings, but at least it is English.

**Comment by iliajie** [comment]
> I am biased because I am here in Blighty (a.k.a UK)

LOL, you see! 😆 

> However you spell color/colour use that language format. Which I am guessing is US.

Perhaps, it might be reasonable and fair to favor US spelling since technology has significantly evolved in the US?

**Comment by shoulders** [comment]
![image](https://github.com/virtualmin/virtualmin-gpl/assets/319997/f07f4e0a-7994-4738-b554-6df5a75f2ef8)

Favor vs Favour

**Comment by iliajie** [comment]
> Favor vs Favour

No, "favor" is actually the American spelling. The British spelling is "favour." 😅

**Comment by jcameron** [comment]
Coming back to the original bug, I think we should only list one of the two spellings in the help output..

**Comment by iliajie** [comment]
> I think we should only list one of the two spellings in the help output

+1.

**Comment by iliaross** [comment]
Reopened, as this was closed by an automated cleanup run. @shoulders, sorry for the noise, and thanks for the contribution. We definitely didn't mean any disrespect.

If it turns out the issue is already resolved for you, you're welcome to close it.
